.reek

```diff
@@ -28,6 +28,7 @@ UnusedPrivateMethod:
     - ActiveJob::Logging::LogSubscriber
     - Users::PhoneConfirmationController
 UtilityFunction:
+  public_methods_only: true
```

Comment:
Yes, that was my first instinct too, to make them class methods. I ended up in gymnastics until I just decided to turn off that check.